src/parser/jobs/rdm/modules/MovementSkills.js

```diff
@@ -0,0 +1,205 @@
+import {t} from '@lingui/macro'
+import {Trans} from '@lingui/react'
+import React, {Fragment} from 'react'
+import {Accordion, Message} from 'semantic-ui-react'
+
+import ACTIONS from 'data/ACTIONS'
+import Rotation from 'components/ui/Rotation'
+import Module from 'parser/core/Module'
+import {Rule, Requirement} from 'parser/core/modules/Checklist'
+import {ActionLink} from 'components/ui/DbLink'
+import {getDataBy} from 'data'
+
+const CACS_PER_MANAFICATION = 3
+const DISPS_PER_MANAFICATION = 3
+const ENGAGEMENTS_PER_MANAFICATION = 4
+//This is due to the fact that manafication happens after your first cac/disp
+const GRACE_FOR_PULL = 2
+
+export default class MovementSkills extends Module {
+	static handle = 'movementskills'
+	static title = t('rdm.movementskills.title')`Movement Skills`
+	static dependencies = [
+		'checklist',
+	]
+
+	_history = []
+	_last_manafic = {
+		timestamp: 0,
+		cac: 0,
+		disp: 0,
+		engagement: 0,
+		events: [],
+	}
+
+	constructor(...args) {
+		super(...args)
+		//Default Target to hit
+		this.target = 95
```

Comment:
can move this and the description to the checklist so it's not polluting the class interface